libcontainer/capabilities/capabilities.go

```diff
@@ -14,24 +15,26 @@ import (
 const allCapabilityTypes = capability.CAPS | capability.BOUNDING | capability.AMBIENT
 
 var (
-	capabilityMap map[string]capability.Cap
-	capTypes      = []capability.CapType{
+	capTypes = []capability.CapType{
 		capability.BOUNDING,
 		capability.PERMITTED,
 		capability.INHERITABLE,
 		capability.EFFECTIVE,
 		capability.AMBIENT,
 	}
+	capMap = sync.OnceValue(initCapMap)
 )
 
-func init() {
-	capabilityMap = make(map[string]capability.Cap, capability.CAP_LAST_CAP+1)
+// Do not call directly, use capMap instead.
+func initCapMap() map[string]capability.Cap {
```

Comment:
Alternatively if you want to avoid the comment, could just inline it I guess;

```go
capMap = sync.OnceValue(func() map[string]capability.Cap {
...
})
```

Reply:
I wanted to minimize diff, but you're right, it's not a huge chunk of code anyway.

Reply:
Yeah, it's a lateral move, so diff isn't horrible (probably would've been worse / harder to read when done in a follow-up after the comment was added)